include/dlaf/matrix/tile.h

```diff
@@ -35,6 +35,81 @@ struct ContinuationException final : public std::runtime_error {
 };
 
 namespace matrix {
+namespace internal {
+
+template <class T, Device device>
+class TileData {
```

Comment:
Just checking my understanding: `TileData` is sort of the "dumb" version of `Tile` that doesn't know anything about dependencies, correct?

Reply:
Yes